lib/evaluate/intrinsics.cc

```diff
@@ -714,10 +714,24 @@ static const IntrinsicInterface genericIntrinsicFunction[]{
 // Tables 16.2 and 16.3 in Fortran 2018.  The "unrestricted" functions
 // in Table 16.2 can be used as actual arguments, PROCEDURE() interfaces,
 // and procedure pointer targets.
+// Note that the restricted conversion functions dcmplx, dreal, float, idint,
+// ifix and sngl are extended to accept any argument kind because this a common
```

Comment:
"this is a common Fortran behavior" rather than "this a common Fortran behavior" (missing the "is").

Reply:
Thanks